public/app/features/explore/ExploreGraphPanel.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { PureComponent } from 'react';
```

Comment:
This is the Explore version of GraphPanel, don't agree totally with the name. Any suggestions very appreciated.